WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -239,18 +253,103 @@ public void onCreate(Bundle savedInstanceState) {
             confirmInstallPluginForAutomatedTransfer();
         }
 
+        if (isShowingDomainCreditCheckProgress) {
+            showDomainCreditsCheckProgressDialog();
+        }
+
         if (mIsShowingAutomatedTransferProgress) {
             showAutomatedTransferProgressDialog();
         }
     }
 
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onPlansFetched(OnPlansFetched event) {
+        if (mCheckingDomainCreditsProgressDialog == null || !mCheckingDomainCreditsProgressDialog.isShowing()) {
+            AppLog.v(T.PLANS, "User cancelled domain credit checking. Ignoring the result.");
```

Comment:
Nitpick: I'd probably prefer "warning" severity as it may also happen due to a bug and it'll be easier to find the source of the bug. Wdyt?